consumers/consume-data.md

```diff
@@ -5,6 +5,7 @@ In this model, Pyth Network does not continuously update the price feeds on most
 Instead, Pyth prices are streamed off-chain via the Wormhole cross-chain messaging protocol.
 Each supported blockchain hosts a Pyth program that supports a permissionless price update operation; this operation accepts a Wormhole message, verifies its content, and updates an on-chain price stored in the program.
 Users of Pyth are responsible for invoking this permissionless update when they wish to use Pyth price.
+For a discussion of the costs and benefits of this model, see [_On-Demand Updates_](on-demand.md).
```

Comment:
Maybe instead of explaining it twice, just link to the on-demand page at the top here?

> Pyth uses an on-demand price update model, where users are responsible for posting price updates on-chain when needed. Please read [_On-Demand Updates_](on-demand.md) to find out why we have chosen this model, and what this means for you.